src/Client/Web/Bit.Client.Web.BlazorUI/Components/Links/BitLink.razor

```diff
@@ -10,16 +10,27 @@
     {
```

Comment:
all the code formattings in this file are changed which is not correct, plz fix this issue manually.